src/lua/bpf/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2016, Marek Vavrusa <marek@vavrusa.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
Not at all, happy to go with apache2. I'll remove the LICENSE file and update rockspec for the next iteration. Do you want a license header in each file or is it enough when it's covered by the `LICENSE.txt` ?